src/components/Nav/Nav.js

```diff
@@ -1,7 +1,102 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import SearchBar from './SearchBar/SearchBar';
 
 const Nav = () => {
-  return <div></div>;
+  return (
+    <Navbar>
+      <header>
+        <nav>
+          <Logo>
+            <Link to="/">
+              RIDIBOOKSL<span>│BOOKSL</span>
+            </Link>
+          </Logo>
+          <Sign>
+            <Link to="/">회원가입</Link>
+            <Link to="/">로그인</Link>
+          </Sign>
+          <SearchBar />
+        </nav>
+        <ul />
+      </header>
+    </Navbar>
+  );
 };
 
+const Navbar = styled.div`
+  width: 100%;
+  background-color: #1f8ce6;
+  color: white;
+
+  header {
+    max-width: 1000px;
+    margin: 0 auto;
+
+    nav {
+      display: flex;
+      align-items: center;
+      padding: 16px 16px 24px 20px;
+    }
+  }
+
+  a {
+    text-decoration: none;
+  }
```

Comment:
`Link`도 컴포넌트이기 때문에, styled의 인자로 넘겨서 다음과 같이 styled component로 만들어 사용할 수도 있습니다.

```suggestion
const Menu = styled(Link)`
  text-decoration: none;
`
```